### ISSUE TITLE
No SQS queues are returned when using the CLI

### ISSUE THREAD

**Issue by dnicolson** [iann0036/former2#146]
The following command fails to generate any resources with version 0.2.55:

`former2 generate --services "SQS" --output-cloudformation "cfn.yaml"`

It appears to be related to the [listQueues](https://github.com/iann0036/former2/blob/590340f/js/services/sqs.js#L68) call which does not include the `QueueUrls` key.

The following code correctly gets the queue URLs:
```
const AWS = require('aws-sdk')
const sqs = new AWS.SQS({region: 'eu-west-1'});

sqs.listQueues({}, (err, data) => {
  console.log(data.QueueUrls);
});
```

The web version also correctly displays the queues:
https://former2.com/#section-applicationintegration-sqs

**Comment by iann0036** [comment]
Hey @dnicolson,

Thanks for reaching out. There may be an issue with this release where the default region isn't being read. Could you confirm whether explicitly setting the region (e.g. `--region us-west-2`) solves your issue?

**Comment by dnicolson** [comment]
Thanks for the quick response. Setting the region explicitly worked. I guess reading from `~/.aws/config` may be a solution?

**Comment by iann0036** [comment]
Thanks for confirming. I've pushed a fix in 0.2.56, hopefully that will load appropriately from the local config file.

**Comment by dnicolson** [comment]
Thanks for the quick fix, I can confirm it works.
